pom.xml

```diff
@@ -157,6 +164,11 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
See above comment.  I don't think this is needed if we bring in the `junit-jupiter-engine` dependency.

Reply:
Fixed. Thanks.